client/src/data/languages.ts

```diff
@@ -0,0 +1,114 @@
+export const LANGUAGES = [
+  {
+    id: 'en',
+    name: 'English',
+  },
+  {
+    id: 'cn',
+    name: 'Chinese',
+  },
+  {
+    id: 'hi',
+    name: 'Hindi',
+  },
+  {
+    id: 'es',
+    name: 'Spanish',
+  },
+  {
+    id: 'fr',
+    name: 'French',
+  },
+  {
+    id: 'ar',
+    name: 'Arabic',
+  },
+  {
+    id: 'bn',
+    name: 'Bengali',
+  },
+  {
+    id: 'ru',
+    name: 'Russian',
+  },
+  {
+    id: 'pt',
+    name: 'Portuguese',
+  },
+  {
+    id: 'id',
+    name: 'Indonesian',
+  },
+  {
+    id: 'ur',
+    name: 'Urdu',
+  },
+  {
+    id: 'ja',
+    name: 'Japanese',
+  },
+  {
+    id: 'de',
+    name: 'German',
+  },
+  {
+    id: 'pa',
+    name: 'Punjabi',
+  },
+  {
+    id: 'te',
+    name: 'Telugu',
+  },
+  {
+    id: 'tr',
+    name: 'Turkish',
+  },
+  {
+    id: 'ko',
+    name: 'Korean',
+  },
+  {
+    id: 'mr',
+    name: 'Marathi',
+  },
+  {
+    id: 'ky',
+    name: 'Kyrgyz',
+  },
+  {
+    id: 'kz',
+    name: 'Kazakh',
+  },
+  {
+    id: 'uz',
+    name: 'Uzbek',
+  },
+  {
+    id: 'ka',
+    name: 'Georgian',
+  },
+  {
+    id: 'pl',
+    name: 'Polish',
+  },
+  {
+    id: 'lt',
+    name: 'Lithuanian',
+  },
+  {
+    id: 'lv',
+    name: 'Latvian',
+  },
+  {
+    id: 'be',
+    name: 'Belarusian',
+  },
+  {
+    id: 'uk',
+    name: 'Ukrainian',
+  },
+].sort((a, b) => {
+  if (a.id > b.id) return 1;
+  if (a.id < b.id) return -1;
+  return 0;
+});
```

Comment:
- It's a good idea to use a constant variable, as it's a read-only value.
- Give the constant variable a descriptive name, such as AVAILABLE_LANGUAGES or SUPPORTED_LANGUAGES.
- It's also a good practice to add a type to this constant variable, so that it's clear what type of data it contains.
- It's also better to use a enum for this kind of data as it makes it more readable and easy to maintain.

```
export enum AVAILABLE_LANGUAGES {
    EN = 'English',
    CN = 'Chinese',
    HI = 'Hindi',
    ES = 'Spanish',
    FR = 'French',
    AR = 'Arabic',
    BN = 'Bengali',
    RU = 'Russian',
    PT = 'Portuguese',
    ID = 'Indonesian',
    UR = 'Urdu',
    JA = 'Japanese',
    DE = 'German',
    PA = 'Punjabi',
    TE = 'Telugu',
    TR = 'Turkish',
    KO = 'Korean',
    MR = 'Marathi',
    KY = 'Kyrgyz',
    KZ = 'Kazakh',
    UZ = 'Uzbek',
    KA = 'Georgian',
    PL = 'Polish',
    LT = 'Lithuanian',
    LV = 'Latvian',
    BE = 'Belarusian',
    UK = 'Ukrainian',
}

```

Reply:
switched to enum on back, could you check it?